geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractIncrIntegrationTest.java

```diff
@@ -104,7 +104,7 @@ public void testIncr_whenWrongType_shouldReturnError() {
     try {
       jedis.incr(key);
     } catch (JedisDataException e) {
-      assertThat(e.getMessage()).contains(RedisConstants.ERROR_NOT_INTEGER);
+      assertThat(e.getMessage()).contains("out of range");
```

Comment:
If our messages are out of sync with native redis shouldn't we be seeing our native acceptance tests failing on this branch? It looks like they all passed. Does this indicate something is wrong with our native redis testing?

Reply:
This one was not out of sync until this PR.  The reason it still passes is because it was changed to only check that the error message contains "out of range" rather than the full message "value is not an integer or out of range".  I'm sure there are other messages throughout our code that are not the same as Redis, but we haven't found them yet because the commands are not fully tested.